lib/dai-plugin-mcd/src/CdpType.js

```diff
@@ -44,10 +44,11 @@ export default class CdpType {
   }
 
   async getPrice() {
-    const { spot } = await this.ilkInfo('pit');
+    const val = await this._web3().eth.getStorageAt(this._pipAddress(), 2);
```

Comment:
instead of reaching into the Web3Service's internals, please [expose the getStorageAt method](https://github.com/makerdao/dai.js/blob/dev/src/eth/Web3Service.js#L88)